CHANGELOG.md

```diff
@@ -127,10 +127,41 @@ the APIs in these libraries are stable, and are ready for production use.
 - [Alloy DB](/google/cloud/alloydb/README.md)
 - [API Keys](/google/cloud/apikeys/README.md)
 
+### [BigQuery](/google/cloud/bigquery/README.md)
+
+- fix: retries for GetIamPolicy, TestIamPermissions ([#10957](https://github.com/googleapis/google-cloud-cpp/pull/10957))
+
+### [Bigtable](/google/cloud/bigtable/README.md)
+
+- fix: retries for GetIamPolicy, TestIamPermissions ([#10957](https://github.com/googleapis/google-cloud-cpp/pull/10957))
+- fix(bigtable): retries for CheckConsistency / AsyncWaitForConsistency ([#10955](https://github.com/googleapis/google-cloud-cpp/pull/10955))
+- docs(bigtable): clean up CreateTable sample ([#10844](https://github.com/googleapis/google-cloud-cpp/pull/10844))
+
+### [Cloud Asset](/google/cloud/asset/README.md)
+
+- feat(generator): prepare comment output for Fedora:37 ([#10749](https://github.com/googleapis/google-cloud-cpp/pull/10749))
```

Comment:
I think I would not highlight this.

Reply:
But I don't feel too strongly about it.

Reply:
Right you are ... I removed this from other places, but missed this one.  Will do.